tensilelite/Tensile/Contractions.py

```diff
@@ -339,6 +348,7 @@ def predicates(self, includeBatch=False, includeOperation=False, includeType=Fal
                 predicates.append(ProblemPredicate("BetaZero"))
             predicates.append(ProblemPredicate("BiasDataTypeWhiteList", value=self.biasDataTypeWhiteList))
             predicates.append(ProblemPredicate("BiasSrcWhiteList", value=self.biasSrcWhiteList))
+            predicates.append(ProblemPredicate("AmaxDCheck", value=self.outputAmaxD))
             if self.activationType == 'all':
```

Comment:
We have non-AmaxD and AmaxD kernels now, so we add the predicate to distinguish the two different kernels